Admin/ORM/MediaAdmin.php

```diff
@@ -36,7 +36,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('context', null, array(
                 'show_filter' => $this->getPersistentParameter('hide_context') !== true,
             ), 'choice', $options)
-            ->add('category', null, array(
+            ->add('category', 'doctrine_orm_choice', array(
```

Comment:
What if the orm is not in use? I had a quick look at the bundle, it seems you can choose to use categories as documents. Also? Why did it work before? I think it worked because the type guessers translated `null` to `doctrine_orm_choice` or something like that if you have the orm installed, am I right? If yes, maybe the right solution would be to get the guessers to work again.